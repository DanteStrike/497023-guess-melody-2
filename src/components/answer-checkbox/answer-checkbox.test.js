import React from "react";
import renderer from "react-test-renderer";
import AnswerCheckbox from "./answer-checkbox.jsx";

it(`Should render correctly AnswerCheckBox component`, () => {
  const component = renderer
    .create(
        <AnswerCheckbox
          id={1}
          value={`checkbox`}
          checked={false}
          onChange={() => {}}
        />
    ).toJSON();

  expect(component).toMatchSnapshot();
});
