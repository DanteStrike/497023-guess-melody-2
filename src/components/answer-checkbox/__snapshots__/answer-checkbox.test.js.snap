// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly AnswerCheckBox component 1`] = `
Array [
  <input
    checked={false}
    className="game__input visually-hidden"
    id="answer-1"
    name="answer"
    onChange={[Function]}
    type="checkbox"
    value="checkbox"
  />,
  <label
    className="game__check"
    htmlFor="answer-1"
  >
    Отметить
  </label>,
]
`;
